### DISCUSSION TITLE
Opening a new Terminal session (Ubuntu 20.04) removes Homebrew from PATH

### DISCUSSION THREAD

**Discussion by IceColdTray** [Homebrew/discussions#2203] [Getting started]
trying to execute `brew help` in a new terminal session doesn't function and produces this error
```
Command 'brew' not found, did you mean:

command 'brec' from deb bplay (0.991-10build1)
command 'qbrew' from deb qbrew (0.4.1-8build1)
```

More info here: https://github.com/Homebrew/brew/issues/12128

**Comment by SMillerDev** [comment]
Check your `~/.profile` to see if `eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"` is actually in there.

**Reply by IceColdTray** [reply]
![image](https://user-images.githubusercontent.com/87967287/134920585-9ee8bba0-f6f9-40fa-9613-88eb538033a7.png)
Yep.

**Reply by SMillerDev** [reply]
Seeing the PATH you posted in the issue you're using a shell that doesn't actually follow conventions and loads `~/.profile`. Try and put this in the configuration file for the shell you're using.

**Reply by IceColdTray** [reply]
How would I do that if you don't mind asking?

**Reply by SMillerDev** [reply]
That really depends on the shell that you're using.

**Comment by elchinzadeh** [comment]
You can add custom alias to `~/.bashrc`:
```alias brew='/home/linuxbrew/.linuxbrew/bin/brew'```
It worked for me
